exercises/2-attack-of-the-pipelines/README.md

```diff
@@ -291,175 +234,88 @@ where the following are the important things:
+1. On your terminal navigate to the root of the `todolist` application. The app contains a hidden folder called `.openshift-applier`. Move into this `.openshift-applier` directory and you should see a familiar looking directory structure for an Ansible playbook.
 ```
-.openshift-applier
 ├── README.md
-├── apply.yml
+├── apply.retry
 ├── inventory
-│   ├── group_vars
-│   │   └── all.yml
-│   └── hosts
+│   ├── group_vars
+│   │   └── all.yml
+│   └── hosts
 ├── params
-│   ├── build
-│   ├── dev
-│   ├── ocp-pipeline
-│   └── test
+│   └── ocp-pipeline
 ├── requirements.yml
+├── roles
+├── site.yml
 └── templates
+    ├── mongodb.yml
     ├── ocp-pipeline.yml
-    ├── todolist-fe-build.yml
-    └── todolist-fe-deploy.yml
+    ├── todolist-build.yml
+    └── todolist-deploy.yml
 ```
 with the following
     * the `apply.yml` file is the entrypoint.
     * the `inventory` contains the objects to populate the cluster with.
     * the `params` contains the variables we'll apply to the `templates`
     * the `templates` required by the app. These include the Build, Deploy configs as well as the services, health checks, and other app definitions.
 
-2. There are a few updates to these manifests we need to make before applying the cluster content. In the `apply.yml` update the namespace `<YOUR_NAME>` variables accordingly.
+2. There are a few updates to these manifests we need to make before applying the cluster content. In the `site.yml` update the namespace `<YOUR_NAME>` variable accordingly.
 ```yaml
-    ci_cd_namespace: donal-ci-cd
-    dev_namespace: donal-dev
-    test_namespace: donal-test
-```
-
-3. In the `params` folder update the `dev` and `test` files with the correct `<YOUR_NAME>` as you've done above. Example for the `dev` file:
-```bash
-PIPELINES_NAMESPACE=donal-ci-cd
-NAME=todolist-fe
-DEPLOYER_USER=jenkins
-APP_TAG=latest
-NAMESPACE=donal-dev
+  vars:
+    namespace_prefix: '<YOUR_NAME>'
+    ci_cd_namespace: '{{ namespace_prefix }}-ci-cd'
+    dev_namespace: '{{ namespace_prefix }}-dev'
+    test_namespace: '{{ namespace_prefix }}-test'
 ```
 
-4. With those changes in place we can now run the playbook. First install the `openshift-applier` dependency and then run the playbook (from the `.openshift-applier` directory). This will populate the cluster with all the config needed for the front end app.
-
-<p class="tip">
-NOTE - Microsoft Windows users, run the *oc* and *ansible* commands inside the *do500-toolbox* container from the appropriate folder.
-</p>
+1. With those changes in place we can now run the playbook. First install the `openshift-applier` dependency and then run the playbook (from the `.openshift-applier` directory). This will populate the cluster with all the config needed for the front end app.
```

Comment:
Maybe clarify by adding with ansible-galaxy - i.e.: `First install the openshift-applier dependency, using the ansible-galaxy tool, and then...`